api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -131,15 +124,11 @@ public void checkFreeTierBillingUsage_exceedsDollarThresholds() throws Messaging
     final DbUser user = createUser(SINGLE_WORKSPACE_TEST_USER);
     createWorkspace(user, SINGLE_WORKSPACE_TEST_PROJECT);
 
-    final short daysLimit = 1000;
-    workbenchConfig.billing.defaultFreeCreditsDaysLimit = daysLimit;
-
     // set an arbitrary registration time
```

Comment:
Free tier alerting for cost has (or had?) a dependency on registration time which is no longer appropriate.  Removing this block would be a good test to see if we still have that.  IIRC there are other similar blocks in this file, which we should also remove.

Reply:
We _do_ also use it to make sure we don't cost-check every single user, only users that are already registered.